### PR TITLE
Normalize file names

### LINKED ISSUE #88: Normalize file names
I suggest `files-like-this.js` and not `FilesLikeThis.js` nor `files_like_this.js`

### PR DESCRIPTION
**Pull request by yeskunall**
🙌🏽🙌🏽🙌🏽

~~**PS**: `lib/electron.js` shows up as a new file because git ignores case-sensitive file names by default. So, I had to `git config core.ignorecase false` in order for git to recognize the change. Hope that's OK 😅~~

The above no longer applies. 🙊

<hr />
Closes #88

### CODE REVIEW

**Comment by vweevers** [comment]
Perhaps rename `electron.js` to `electron-browser.js`, so that they all end in `-browser.js`. Two birds with one stone!

**Comment by ralphtheninja** [comment]
`flatten-browser.js`, `scout-browser.js` etc are a bit confusing now, maybe put those type of functions in e.g. `util.js`? we don't have to do this now but maybe in a later PR

**Comment by vweevers** [comment]
> flatten-browser.js, scout-browser.js etc are a bit confusing now, maybe put those type of functions in e.g. util.js? we don't have to do this now but maybe in a later PR

Or rename them to `flatten-browserlist.js` and something like `sauce-browserlist.js`, respectively.

**Comment by ralphtheninja** [comment]
Or that :)

File: lib/airtap.js [hunk 1]
```diff
@@ -6,9 +6,9 @@ var EventEmitter = require('events').EventEmitter
 
 var createControlServer = require('./control-app')
 var setup_test_instance = require('./setup')
-var SauceBrowser = require('./SauceBrowser')
-var PhantomBrowser = require('./PhantomBrowser')
-var Electron = require('./Electron')
+var SauceBrowser = require('./sauce-browserlist')
+var PhantomBrowser = require('./phantom-browserlist')
+var Electron = require('./electron-browserlist')
```
Comment (ralphtheninja) [inline lib/airtap.js]: `sauce-browser`
`phantom-browser`
`electron-browser`

Comment (yeskunall) [reply lib/airtap.js]: Are you saying the variable name should follow the above pattern as well? Or are you saying the `require` statements are wrong? Sorry I'm a bit confused 😓

Comment (ralphtheninja) [reply lib/airtap.js]: The require statements are wrong. Those wouldn't be valid variables names :wink:

**Comment by ralphtheninja** [comment]
> Or rename them to flatten-browserlist.js and something like sauce-browserlist.js, respectively.

@yeskunall I think @vweevers meant `flatten-browserlist.js` and `scout-browserlist.js` .. not `sauce-browserlist.js`

File: lib/airtap.js (hunk 1 above)
Comment (ralphtheninja) [reply lib/airtap.js]: Maybe also rename `Electron` to `ElectronBrowser` for symmetry.

**Comment by yeskunall** [comment]
All right, that makes a lot more sense to me now. 🙊 thanks for clearing that up @ralphtheninja 😅

Just had one question: why is it called `scout-browserlist` when really it's hitting up Sauce Labs to fetch a list of available browsers? Or does "scout" here literally mean exactly that? That it's "scouting" for available browsers? 😋

**Comment by ralphtheninja** [comment]
The name is odd though. We can pick a better name if you like. What about `get-saucelabs-browsers.js`? :)

**Comment by yeskunall** [comment]
Thanks for the feedback! I'll merge this soon enough! 😁

**Comment by ralphtheninja** [comment]
> Thanks for the feedback! I'll merge this soon enough! grin

Lets wait for the tests to pass. Then squash the commits please.

**Pull request merged**